packages/edit-site/src/components/page-library/search-items.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * External dependencies
+ */
+import removeAccents from 'remove-accents';
+import { noCase } from 'change-case';
```

Comment:
These dependencies were added to the `edit-site` package to mirror the block editor's filtering in the inserter. Given the dependencies were already within the overall project we didn't see it as a blocker to add them here.